src/clims/services/wrapper.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+
+class WrapperMixin(object):
+    """
+    This mixin can be used to provide wrapping methods to a service class which
+    handles translating models of `ExtensibleModel` to their specific `ExtensibleBase`
+    implementations.
+    The implementing class needs to provide `_archetype_version`, `_archetype`, and
+    `_archetype_base`.
+    """
+
+    _archetype_version = None
```

Comment:
Perhaps it could be indicated that these are class variables and not instances of some class?